README.md

```diff
@@ -9,7 +9,10 @@ Welcome to a LLM Response Analysis Framework! This tool is designed to dive deep
 ## Features
 
 - **Dynamic LLM Integration**
-Seamlessly connect with various LLM providers and models to fetch responses using a flexible architecture.
+Seamlessly connect with various LLM providers and models to fetch responses using a flexible architecture. Following integrations are available.
+   - Openai
+   - Groq
```

Comment:
question (a naive question ;) ): as a user, what do we need to do to set up Groq?

Reply:
have a GROQ_API_KEY same as OPENAI_API_KEY setup

Reply:
add some instructions like `export GROQ_API_KEY=<your-api-key-here>` to this bit https://github.com/thompsonson/det?tab=readme-ov-file#configuration 

ps. I definitely see the intention of the check, let me know what you think about a `det --check-config` style command.

Reply:
yes sure will add. 

yes I've reverted on comment in above thread for config-check.

Reply:
added in docs